rollbar/contrib/pyramid/__init__.py

```diff
@@ -40,31 +49,25 @@ def parse_settings(settings):
 def rollbar_tween_factory(pyramid_handler, registry):
     settings = parse_settings(registry.settings)
 
-    whitelist = ()
-    blacklist = (WSGIHTTPException,)
-
     def rollbar_tween(request):
         # for testing out the integration
         try:
             if (settings.get('allow_test', 'true') == 'true' and
                     request.GET.get('pyramid_rollbar_test') == 'true'):
                 try:
                     raise Exception("pyramid_rollbar test exception")
-                except:
-                    handle_error(settings, request)
+                except Exception as exc:
```

Comment:
Technically, `except Exception as exc:` is different from `except:` but I am not sure we wanted to catch those special cases (BaseException, SystemExit, KeyboardInterrupt, etc) in the first place.